### PR TITLE
Removing ignition

### PR DESCRIPTION
**Pull request by nkoenig**
# 🎉 New feature

Removing "ignition"

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: include/gz/sim/components/Model.hh [hunk 1]
```diff
@@ -58,8 +58,8 @@ namespace serializers
         {
           // Skip serializing models with //pose/@relative_to attribute
           // since deserialization will fail. This could be a nested model.
-          // see https://github.com/ignitionrobotics/ign-gazebo/issues/1071
-          // Once https://github.com/ignitionrobotics/sdformat/issues/820 is
+          // see https://github.com/gazebosim/gz-gazebo/issues/1071
```
Comment (ahcorde) [inline include/gz/sim/components/Model.hh:61]:
```suggestion
          // see https://github.com/gazebosim/gz-sim/issues/1071
```

**Review by ahcorde — COMMENTED** [review]
typo

File: src/SystemLoader.cc [hunk 2]
```diff
@@ -63,16 +63,7 @@ class gz::sim::SystemLoaderPrivate
   public: bool InstantiateSystemPlugin(const sdf::Plugin &_sdfPlugin,
               gz::plugin::PluginPtr &_gzPlugin)
   {
-    // Deprecated: accept ignition-gazebo-prefixed systems
```
Comment (azeey) [inline src/SystemLoader.cc:66]: We sort of discussed this in the weekly meeting. Removing this means any existing SDFormat file that has the `ignition-gazebo` plugins will not work in Harmonic. Are we okay with that?

File: src/systems/air_speed/AirSpeed.cc [hunk 3]
```diff
@@ -307,4 +307,4 @@ GZ_ADD_PLUGIN(AirSpeed, System,
 GZ_ADD_PLUGIN_ALIAS(AirSpeed, "gz::sim::systems::AirSpeed")
 
 // TODO(CH3): Deprecated, remove on version 8
-GZ_ADD_PLUGIN_ALIAS(AirSpeed, "ignition::gazebo::systems::AirSpeed")
+GZ_ADD_PLUGIN_ALIAS(AirSpeed, "gz::sim::systems::AirSpeed")
```
Comment (azeey) [inline src/systems/air_speed/AirSpeed.cc:310]: This just needs to be removed.

File: src/systems/log/LogPlayback.cc [hunk 4]
```diff
@@ -510,9 +510,9 @@ void LogPlayback::Update(const UpdateInfo &_info, EntityComponentManager &_ecm)
   {
     auto msgType = iter->Type();
 
-    // Support ignition.msgs for backwards compatibility, don't remove on tock
+    // Support gz.msgs for backwards compatibility, don't remove on tock
```
Comment (azeey) [inline src/systems/log/LogPlayback.cc:513]: I think it worth keeping backward compatibility here.

**Review by azeey — COMMENTED** [review]
Looks good. My main concern is if we should remove support for using "ignition" in SDF files. Unlike code, those files are meant to work with multiple versions of Gazebo.

File: src/systems/air_speed/AirSpeed.cc (hunk 3 above)
Comment (nkoenig) [reply src/systems/air_speed/AirSpeed.cc]: Removed in [fa23fcc](https://github.com/gazebosim/gz-sim/pull/2055/commits/fa23fcce063854af27c64421fb4d980be169514a)

File: src/systems/log/LogPlayback.cc (hunk 4 above)
Comment (nkoenig) [reply src/systems/log/LogPlayback.cc]: Added back in [d3b2dd1](https://github.com/gazebosim/gz-sim/pull/2055/commits/d3b2dd1ac8f5321bb958e3a73e7de18496cbccce)

**Comment by mjcarroll** [comment]
This now needs #2087 to build

**Comment by nkoenig** [comment]
> Looks good. My main concern is if we should remove support for using "ignition" in SDF files. Unlike code, those files are meant to work with multiple versions of Gazebo.

I believe we have followed the tick-tock approach where deprecation warnings have been given to users to modify their SDF. Standard practice is to follow the tick with the tock.

**Comment by azeey** [comment]
> I believe we have followed the tick-tock approach where deprecation warnings have been given to users to modify their SDF. Standard practice is to follow the tick with the tock.

Fair enough.

**Comment by iche033** [comment]
changes look goo to me. Just need to re-run CI when it's ready.

**Review by azeey — APPROVED** [review]

**Review by mjcarroll — APPROVED** [review]

**Comment by mjcarroll** [comment]
Now blocked on https://github.com/gazebosim/gz-msgs/pull/380 and https://github.com/gazebosim/gz-sim/pull/2103

**Comment by mjcarroll** [comment]
> What are you doing to me?

Jumping on the Windows grenade for you :upside_down_face:

**Comment by mjcarroll** [comment]
It's not actually blocked at this point, Windows CI isn't going to pass until those two PRs are in.  One for gz-msgs generation stuff, one for disabling pybind on windows which hadn't been forward ported yet.

I'm good with merging with those caveats.

**Comment by nkoenig** [comment]
A windows grenade seems like the most ineffectual grenade ever. Let me know when the grenade is done updating. 

I'll wait...

**Comment by mjcarroll** [comment]
CI Except Windows looks okay

**Pull request merged**